openedx/adg/lms/applications/constants.py

```diff
@@ -4,3 +4,7 @@
 
 LOGO_IMAGE_MAX_SIZE = 200 * 1024
 ALLOWED_LOGO_EXTENSIONS = ('png', 'jpg', 'svg')
+
+RESUME_AND_COVER_LETTER = 'RESUME & COVER LETTER'
```

Comment:
How about just leaving a comment at the top for all fieldset titles. Because appending 'FIELDSET_TITLE' with each of these string constants may look a bit redundant, e.g. in:

`fieldset_title = FIELDSET_TITLE_RESUME_AND_COVER_LETTER`